.github/workflows/test.yml

```diff
@@ -0,0 +1,134 @@
+name: Test
+
+on:
+  pull_request:
+    branches:
+      - master
+      - maintenance/**
+  push:
+    branches:
+      - master
+      - maintenance/**
+    tags:
+      - "v*"
+
+env:
+  PYTHONUNBUFFERED: "1"
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: ubuntu-18.04
+            python: "2.7"
+            toxenv: cp27
+            continue-on-error: false
+          - os: ubuntu-18.04
+            python: "3.4"
+            toxenv: cp34
+            continue-on-error: false
+          - os: ubuntu-18.04
+            python: "3.5"
+            toxenv: cp35
+            continue-on-error: false
+          - os: ubuntu-18.04
+            python: "3.6"
+            toxenv: cp36
+            continue-on-error: false
+          - os: ubuntu-18.04
+            python: "3.7"
+            toxenv: cp37
+            continue-on-error: false
+          - os: ubuntu-18.04
+            python: "3.8"
+            toxenv: cp38
+            continue-on-error: false
+          - os: ubuntu-18.04
+            python: "3.9"
+            toxenv: cp39
+            continue-on-error: false
+
+          - os: ubuntu-18.04
+            python: "pypy2"
+            toxenv: pp2
+            continue-on-error: false
+          - os: ubuntu-18.04
+            python: "pypy3"
+            toxenv: pp3
+            continue-on-error: false
+
+          - os: ubuntu-20.04
+            python: "2.7"
+            toxenv: cp27
+            continue-on-error: false
+          - os: ubuntu-20.04
+            python: "3.5"
+            toxenv: cp35
+            continue-on-error: false
+          - os: ubuntu-20.04
+            python: "pypy2"
+            toxenv: pp2
+            continue-on-error: false
+          - os: ubuntu-20.04
+            python: "pypy3"
+            toxenv: pp3
+            continue-on-error: false
+
+          - os: macos-10.15
+            python: "3.5"
+            toxenv: cp35
+            continue-on-error: true
+          - os: macos-11.0
+            python: "3.5"
+            toxenv: cp35
+            continue-on-error: true
+          - os: windows-2019
+            python: "3.5"
+            toxenv: cp35
+            continue-on-error: false
+
+          # 3.10 Alpha
+          - os: ubuntu-18.04
+            python: "3.10.0-alpha.2"
+            toxenv: "cp310"
+            continue-on-error: false
```

Comment:
Should we set this also to true?

Reply:
We can do that if it starts failing in a newer alpha.